org.qtodotxt.qtodotxt.appdata.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2014-2017 Almar Klein -->
+<!-- This is a appdata file for freedesktop software centers. -->
+<!-- Please let me know if this needs changes. -->
+<component>
+  <id type="desktop">org.qtodotxt.qtodotxt.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>BSD-2-Clause</project_license>
+   <releases>
+    <release date="2009-01-09" version="1.7"/>
+  </releases>
+  <name>qtodotxt</name>
```

Comment:
The appdata could use a description, launchable, developer_name, update_contact. 
See the specs here: https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
We also need `launchable` to get this in.

Reply:
bump